charts/elasticsearch/templates/client/es-client-deployment.yaml

```diff
@@ -86,11 +86,11 @@ spec:
       containers:
       - name: es-client
         securityContext:
+        {{- if .Values.securityContext_esclient }}
+{{ toYaml .Values.securityContext_esclient | indent 10 }}
+        {{- else }}
 {{ toYaml .Values.securityContext | indent 10 }}
-          capabilities:
-            add:
-              - IPC_LOCK
-              - SYS_RESOURCE
+        {{- end }}
```

Comment:
Now that we have an if/else, we should add a test to show that the client config overrides the global config.